.travis.yml

```diff
@@ -5,6 +5,7 @@ rvm:
 - 2.1.5
 - 2.2.0
 - 2.3.0
+- jruby-9.1.5.0
```

Comment:
@vishrutshah I think your suggestion is reasonable, we can try using `jruby-head`, so we know that we work on the latest. If we start seeing problems, we can revisit. thanks!